hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -3204,11 +3235,12 @@ void renameTo(final String src, final String dst,
         res = FSDirRenameOp.renameToInt(dir, pc, src, dst, logRetryCache,
             options);
       } finally {
-        writeUnlock(operationName);
+        FileStatus status = res != null ? res.auditStat : null;
+        writeUnlock(operationName,
+            getLockReportInfoSupplier(src, dst, status));
       }
     } catch (AccessControlException e) {
-      logAuditEvent(false, operationName + " (options=" +
-          Arrays.toString(options) + ")", src, dst, null);
+      logAuditEvent(false, operationName, src, dst, null);
```

Comment:
I think you need to rebase, right?

Reply:
Thank you for pointing out this. I modified the patch for the review.